packages/babel-compat-data/data/native-modules.json

```diff
@@ -9,9 +9,10 @@
     "opera": "48",
     "op_mob": "48",
     "safari": "10.1",
-    "ios_saf": "10.3",
+    "ios": "10.3",
```

Comment:
`esmodule: "intersect"` does not work on `ios` targets because it was `ios_saf`.